src/ServicePulse.Host/vue/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "service-pulse",
   "version": "1.0.0",
+  "type": "module",
   "description": "Production monitoring for distributed systems.",
   "main": "index.html",
   "scripts": {
-    "dev": "npm run lint && vite",
-    "build": "npm run lint && vite build",
-    "preview": "npm run lint && vite preview",
-    "test:unit": "npm run lint && vitest --environment jsdom --root src/",
-    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs"
+    "dev": "vite",
+    "build": "vite build",
```

Comment:
don't we want the linter to run before build on the build agent? Or does this now happen through another mechanism?

Reply:
it happens automatically via the [checkers](https://github.com/Particular/ServicePulse/pull/1693/files#diff-17992c31932202118ec9a1ea7633664bfe09ad0cb9e9f67220d47469e3a95043R34) configured in vite.config.ts